tooling/test/sugar.go

```diff
@@ -2,58 +2,143 @@ package test
 
 import (
 	"fmt"
+	"net/url"
+	"strings"
 
 	"github.com/ipfs/gateway-conformance/tooling/check"
 )
 
+// When a test writer uses a URL that contains the example.com domain, we assume they
+// mean "the current subdomain URL". This means that "http://something.ipfs.example.com/path"
+// is rewritten to hit the current tested gateway, which might live under a different name, and
+// different port.
+func TestURLFromSpecURL(testURL string) string {
```

Comment:
We're not using this, right?

Reply:
forgotten dead code, thanks for catching it